app/javascript/src/components/Dashboard/Notes/Note/index.jsx

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line react/jsx-filename-extension
+import React from "react";
+
+import Content from "./Content.jsx";
+import Footer from "./Footer.jsx";
+import Header from "./Header.jsx";
+import Tooltip from "./Tooltip.jsx";
+
+const Note = ({ NOTES_DATA }) =>
+  NOTES_DATA.map(NOTE => (
+    <>
```

Comment:
Why do we need this fragment?

Reply:
Since I wanted spacing between the individual note, I needed a break statement. To enclose individual components and break statement I used this fragment.

Reply:
Got it. Didn't see the `br` tag.